public/locales/de/translation.json

```diff
@@ -0,0 +1,43 @@
+{
+  "intro": {
+    "text": [
+      "In diesem Spiel gibt es zwei Rollen: die investigierende und die lügende Person. Jede Runde wechselt sich diese Rolle.",
+      "Die lügenden Person werden zwei zufällige Wikipedia Artikel präsentiert. Nur einer der beiden Artikel kann gelesen werden.",
+      "Die investigierende Person sieht die Titel beider Artikel und bekommt die möglichkeit, fragen zu den Artikeln zu stellen.",
+      "Wird die investigierende Person herausfinden, welchen Artikel sich die lügende Person ausgedacht hat?"
+    ],
+    "language": "<0>sprache: </0>",
+    "player-one": "<0>spieler eins: </0>",
+    "player-two": "<0>spieler zwei: </0>",
+    "player-identical": "Spieler können nicht den gleichen Namen haben",
+    "start": "Start!",
+    "footer": "Dieses Spiel wurde inspiriert von <2>Two Of These People Are Lying</2> von Tom Scott und Matt Grey."
+  },
+  "game": {
+    "try-again": "try again",
+    "choosing": "Die lügende Person ({{liar}}) ist jetzt am Zug. Entscheide dich für einen der Artikel, um dich über ein Gebiet zu informieren:",
+    "article": "Artikel {{number}}",
+    "done-reading": "Ich bin fertig",
+    "preguessing": "Bitte reiche {{investigator}} das Gerät weiter.",
+    "start-guessing": "raten beginnen",
+    "guessing": "Die investigierende Person ({{investigator}}) ist jetzt am Zug. Du kannst Fragen zu diesen Artikeln stellen. Sobald du eine Vermutung hast welchen Artikel die andere Person gelesen hat, kannst du abstimmen!",
+    "scratchpad": "Hier hast du einen Schmierblock für deine Notizen:",
+    "scratchpad-tips": "Interessante details die sie wussten, haben sie das gerade gelesen? Oder ist das nur Zufall… Schreibe es zur Erinnerung auf.",
+    "investigator-won": "Die investigierende Person, {{ investigator }} hat gewonnen! {{ liar }} hat {{ title }} gelesen",
+    "liar-won": "Die lügende Person, {{ liar }} hat gewonnen! {{ liar }} hat eigentlich {{ title }} gelesen",
```

Comment:
is this Austrian? Google Translate tells me it's `liegende`

Reply:
`liegen` is when you lie down.
`lügen` is when you lie like in this case.